modules/Search/components/Results.tsx

```diff
@@ -1,58 +1,60 @@
-import type { AlgoliaStory } from '@prezly/theme-kit-core';
-import { translations } from '@prezly/theme-kit-intl';
+'use client';
+
+import { translations } from '@prezly/theme-kit-nextjs';
 import classNames from 'classnames';
-import type { Hit as HitType, InfiniteHitsProvided } from 'react-instantsearch-core';
+import type { InfiniteHitsProvided } from 'react-instantsearch-core';
 import { connectInfiniteHits } from 'react-instantsearch-dom';
-import { useIntl } from 'react-intl';
 
-import { Button } from '@/ui';
+import { useIntl } from '@/adapters/client';
+import { Button } from '@/components/Button';
 
 import { useAlgoliaState } from './AlgoliaStateContext';
-import Hit from './Hit';
+import type { Props as HitProps } from './Hit';
+import { Hit } from './Hit';
 
 import styles from './Results.module.scss';
-import containerStyles from '@/modules/InfiniteStories/InfiniteStories.module.scss';
-import listStyles from '@/modules/InfiniteStories/StoriesList.module.scss';
-
-type SearchHit = HitType<{ attributes: AlgoliaStory }>;
-
-function Results({ hits, hasMore, refineNext }: InfiniteHitsProvided<SearchHit>) {
-    const { formatMessage } = useIntl();
-    const { searching: isSearching } = useAlgoliaState();
-
-    return (
-        <div className={containerStyles.container}>
-            <div className={classNames(listStyles.storiesContainer, styles.list)}>
-                {!hits.length && (
-                    <p className={styles.fallbackText}>
+import containerStyles from '@/modules/InfiniteStories/InfiniteStories.module.scss'; // FIXME: Pass this from outside
+import listStyles from '@/modules/InfiniteStories/StoriesList.module.scss'; // FIXME: Find a way to pass this from ouside
```

Comment:
👀

Leaving a comment just so we don't forget about it.

Reply:
Yeah, but this is not for now. We'll address it while switching to the Design System UI components we've prepared.